src/webhooks/pubsub/index.ts

```diff
@@ -360,7 +360,11 @@ export const constructSlackMessage = (
         hasEnoughTimePassedSinceLastNotification &&
         hasEnoughTimePassedSinceIssueCreation;
       const formattedIssues = sortAndFlattenIssuesArray(triageQueueIssues);
-      if (triageQueueIssues.length > 0) {
+      if (
+        triageQueueIssues.length > 0 &&
+        overdueIssues.length === 0 &&
+        actFastIssues.length === 0
+      ) {
```

Comment:
Sorry it's really not obvious to me how this is the fix. 🐭

Approving in the interest of speed but I don't get it.

Reply:
Basically, we first populate the `overdueIssues` and `actFactIssues` list if there are any issues that belong to those respective groups. The `triageQueueIssues` is the last one populated. This changes it so we will **only** add to the triage queue if there are no issues that are overdue or close to being overdue.

Reply:
added a comment

Reply:
ty 84a4f795ff133dda8a439592b59288903f6462df